pages/common/pywal.md

```diff
@@ -0,0 +1,28 @@
+# pywal
+
+> A tool to create colorschemes based on dominant colors of a wallpaper.
+> Read more: https://github.com/dylanaraps/pywal.
+
+- Preview colorscheme:
+
+`wal --preview image.png`
+
+- Create colorscheme:
+
+`wal -i image.png`
```

Comment:
Is there a long form of the `-i` option here? In tldr pages we prefer to use the long form option where possible to aid clarity. The same goes for the later examples too.

Reply:
There is no long form of the `-i` option, along with all the other examples. `--preview` is the only long form option in that example.